samples/net/sockets/echo_server/overlay-ppp.conf

```diff
@@ -1,3 +1,5 @@
+CONFIG_NET_TCP=n
```

Comment:
How does it break the compilation? I'd rather focus on fixing the issue instead of disabling TCP for the sample running with ppp.

Reply:
I agree
I'll try to find a solution if I find some spare time